### ISSUE TITLE
Support for Mic Input On Windows?

### ISSUE THREAD

**Issue by ns96** [project-owner/PeppyMeter.doc#14]
Hi, I am interesting is using PeppyMeter with mic input on windows, but looking through the documentation I can't seem to find any information on doing so. I already got it running on Windows 11 with the the input data being random noise I guess.

**Comment by project-owner** [comment]
Hi, you are probably using a 'noise' data source which shows random data. This data source was created for development and testing purposes. So that the software could be developed either on Windows or any other platform.

The other data source should be used for a real audio volume data. It's called 'pipe'. PeppyMeter reads that volume data from a 'named pipe'. A special ALSA plugin was created to provide that volume data in the 'named pipe'. All these concepts exist only on Linux platforms. I've  heard that there is a concept of the 'named pipe' on Windows as well but I'm not sure how to use it from Python. Also something should provide volume data to that pipe on Windows. I don't know how to do that.

Long story short, PeppyMeter was developed for using on Linux platforms.

**Comment by ns96** [comment]
Ok thanks for clarifying this. I will look into using the http input source and write a small python script to send the sound data to it.
